autoload/codi.vim

```diff
@@ -536,10 +546,8 @@ function! s:codi_handle_done(bufnr, output)
   " Go to target buf
   exe 'keepjumps keepalt buf! '.a:bufnr
   let s:updating = 1
-  let i = s:get_codi('interpreter')
```

Comment:
This is one of the lines causing an issue. `s:get_codi` depends the current buffer. Here, we are at the "target buffer". However, with the refactor, we grab the value for the "codi buffer".